docs/modules/ROOT/pages/faqs.adoc

```diff
@@ -1,83 +1,100 @@
+The source for this FAQ and related CSC documents are co-located with the Cassandra Source Connector repo code, examples, and additional documentation: access the [repo here](https://github.com/riptano/cdcrep).
 
 == Installation
 
-=== Is there a prerequisite version of Java needed for the DataStax Cassandra Data Connector installation?
+=== Is there a prerequisite version of Java needed for the DataStax Cassandra Source Connector installation?
+
+The DataStax Cassandra Source Connector distribution is designed for Java 11. However, because the product releases Docker images, you do not need to install Java (8 or 11) in advance. Java 11 is bundled in the Docker image.
+
+=== How do I install Cassandra Source Connector?
 
-The DataStax Cassandra Data Connector distribution is designed for Java 11. However, because the product releases Docker images, you do not need to install Java (8 or 11) in advance. Java 11 is bundled in the Docker image.
+Download the tar file from the DataStax [Downloads site](https://downloads.datastax.com/#csc). Then follow the xref:install.adoc[install] instructions. The tarball contains the Cassandra Source Connector distribution's NAR file.
 
-=== How do I install Cassandra Data Connector?
+=== How many Cassandra Tables can I track?
 
-Download the tar file from the DataStax [Downloads site](https://downloads.datastax.com/#cdc). Then follow the xref:install.adoc[install] instructions. The tarball contains the Cassandra Data Connector distribution's NAR file.
+As many as your system can handle.  Note that each Cassandra table CDC events must be tracked with a separate Cassandra Source Connector.
 
 == Tasks
 
-=== What task can I perform in the DataStax Cassandra Data Connector Installation Console?
+=== What task can I perform in the DataStax Cassandra Source Connector Installation Console?
 
 From the Installation Console, you can:
 
-* Configure the DataStax Cassandra Data Connector environment
-* View and update the DataStax Cassandra Data Connector version and application status
-* Get the host URL and generated password to launch the DataStax Cassandra Data Connector Admin Console
-* Set up and view Grafana graphs, which display metrics collected by the provided Prometheus Operator
-* Analyze DataStax Cassandra Data Connector to collect information you could provide for Support
-* Sync the license that enables DataStax Cassandra Data Connector software
-* View upstream, midstream, and downstream files used by DataStax Cassandra Data Connector
+* Configure the CSC environment.
+* View and update the CSC version and application status.
+* Retrieve the host URL and generated password to launch the CSC Admin Console
+* Set up and view Grafana graphs, which display metrics collected by the provided Prometheus Operator.
+* Analyze the CSC to collect information you could provide for support functions.
+* Sync the license that enables CSC software.
+* View upstream, midstream, and downstream files used by the CSC.
```

Comment:
Do we have documentation to how to do this? Is this something we plan to add?

Reply:
I believe we should add this in - I don't see any references yet.

@vroyer: This is Open Source software talking to another open source software, what license sync is there?